### PR TITLE
Release Firefox Extension via CI

### PR DESCRIPTION
**Pull request by fmoessbauer**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,3 +53,14 @@ following:
...
+## Maintainers: Create Releases
+
+The creation of public releases is a partially automated process:
+
+1. update code and create release tags: `VERSION=<x.x.x> make release`
+2. push to GitHub: `git push origin main && git push origin v<x.x.x>`
+3. wait for release action to finish (public release is created)
+4. download signed xpi artifact
+5. update release information in `.pages/firefox/updates.json`
+6. commit updated release information to master
```
Comment (bufferoverflow) [inline CONTRIBUTING.md:66]:
```suggestion
6. commit updated release information to main
```

Comment (jan-kiszka) [inline CONTRIBUTING.md:66]: We were discussing if you cannot make the workflow wait for the release to be done, fetch and patch the updates.json automatically. Any blockers for this?

Comment (fmoessbauer) [reply CONTRIBUTING.md:66]: As the wait-for-sign is now implemented, there are no further blockers. But I don't really like the idea that the CI pushes code to the repo. I'll think about it again.

Comment (fmoessbauer) [reply CONTRIBUTING.md]: Finally, we can fully-automatically release new version of the FF extension.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          VERSION=$(echo ${{ github.ref_name }} | cut -c 2-)
+          DIGEST="sha256:$(sha256sum build/linux_entra_sso-${VERSION}.xpi | cut -d ' ' -f 1)"
+          LINK="https://github.com/siemens/linux-entra-sso/releases/download/v${VERSION}/linux_entra_sso-${VERSION}.xpi"
+          jq --arg version "${VERSION}" --arg digest "${DIGEST}" --arg link "${LINK}" \
+            '."addons"."linux-entra-sso@example.com"."updates" += [{"version":$version, "update_link":$link, "update_hash":$digest}]' \
+            .pages/firefox/updates.json \
+            > .pages/firefox/updates.json.tmp && mv .pages/firefox/updates.json.tmp .pages/firefox/updates.json
+
+      - name: commit update manifest for Firefox
+        uses: stefanzweifel/git-auto-commit-action@v5
```
Comment (jan-kiszka) [inline .github/workflows/release.yml:71]: Shouldn't we better pin this dependency to a specific version?

**Review by jan-kiszka — COMMENTED** [review]
Looks good otherwise

File: .github/workflows/release.yml (hunk 2 above)
Comment (fmoessbauer) [reply .github/workflows/release.yml:71]: In the github ecosystem, the versions are usually just pinned to a tag, but not to a specific commit. By that, I want to keep it like that.

**Pull request merged**